android/src/main/java/com/oblador/keychain/DeviceAvailability.kt

```diff
@@ -15,7 +15,7 @@ import androidx.biometric.BiometricManager
 object DeviceAvailability {
 
   fun isStrongboxAvailable(context: Context): Boolean {
-    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
This will actually make the `supportsSecureHardware` return false, but actually it's still hardware backed with TEE